spec/features/crops/crop_wranglers_spec.rb

```diff
@@ -48,7 +48,7 @@
       fill_in 'en_wikipedia_url', with: "http://en.wikipedia.org/wiki/Maize"
       fill_in 'sci_name[1]', with: "planticus maximus"
       click_on 'Save'
-      expect(page).to have_content 'Crop was successfully created'
+      expect(page).to have_content 'crop was successfully created.'
```

Comment:
the flash notices have changed a little because they're from the responders gem now.